### ISSUE TITLE
The terminal process "C:\Windows\System32\cmd.exe /c (type "C:\Users....

### ISSUE THREAD

**Issue by lindorx** [microsoft/vscode-remote-release#5947]
- VSCode Version:1.63.0-insider (all version)
- Local OS Version: windows 11
- Remote OS Version: linux deepin20.2.4
- Remote Extension/Connection Type: SSH/Docker/WSL
- Logs:The terminal process "C:\Windows\System32\cmd.exe /c (type "C:\Users\lindo\AppData\Local\Temp\vscode-linux-multi-line-command-my-297469433.sh" | ssh -T -D 57264 my bash) & exit /b 0" terminated with exit code: 1.
output:
```
[14:10:59.583] Log Level: 2
[14:10:59.587] remote-ssh@0.66.1
[14:10:59.587] win32 x64
[14:10:59.588] SSH Resolver called for "ssh-remote+my", attempt 1
[14:10:59.589] "remote.SSH.useLocalServer": false
[14:10:59.589] "remote.SSH.showLoginTerminal": false
[14:10:59.591] "remote.SSH.remotePlatform": {}
[14:10:59.591] "remote.SSH.path": undefined
[14:10:59.591] "remote.SSH.configFile": undefined
[14:10:59.591] "remote.SSH.useFlock": true
[14:10:59.592] "remote.SSH.lockfilesInTmp": false
[14:10:59.592] "remote.SSH.localServerDownload": auto
[14:10:59.592] "remote.SSH.remoteServerListenOnSocket": false
[14:10:59.592] "remote.SSH.showLoginTerminal": false
[14:10:59.592] "remote.SSH.defaultExtensions": []
[14:10:59.592] "remote.SSH.loglevel": 2
[14:10:59.593] SSH Resolver called for host: my
[14:10:59.593] Setting up SSH remote "my"
[14:10:59.644] Using commit id "c1f2f40bcd81380bf2e286af64becddada57db88" and quality "insider" for server
[14:10:59.648] Install and start server if needed
[14:11:15.463] Checking ssh with "ssh -V"
[14:11:15.507] > OpenSSH_for_Windows_8.1p1, Li
[14:11:15.507] > breSSL 3.0.2

[14:11:15.511] Running script with connection command: ssh -T -D 57264 my bash
[14:11:15.513] Terminal shell path: C:\Windows\System32\cmd.exe
[14:11:16.950] "install" terminal command done
[14:11:16.951] Install terminal quit with output: 
[14:11:16.951] Resolver error: Error: Got bad result from install script
	at Function.Create (c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:429193)
	at Object.t.tryInstall (c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:521466)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:485356
	at async Object.t.withShowDetailsEvent (c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:488706)
	at async Object.t.resolve (c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:486435)
	at async c:\Users\lindo\.vscode-insiders\extensions\ms-vscode-remote.remote-ssh-0.66.1\out\extension.js:1:560057
[14:11:16.954] ------
```

Steps to Reproduce:
I can't reproduce the bug.This occurs after uninstalling adaconda.
It seems that remote uses CMD terminal to execute shell script, but I don't know how to modify terminal shell path.I hope you can provide more convenient tutorials.

Does this issue occur when you try this locally?: Yes/No
Does this issue occur when you try this locally and all extensions are disabled?: Yes/No

**Comment by tanhakabir** [comment]
Does `C:\Windows\System32\cmd.exe` not exist?

And moreover what exactly are you looking to do? Use PowerShell instead?

> This occurs after uninstalling adaconda.
I'm concerned how this affected SSH. Do you know why uninstalling anaconda caused issues?

**Comment by lindorx** [comment]
> 不存在？`C:\Windows\System32\cmd.exe`
> 
> 此外，您到底想做什么？改用 PowerShell？
> 
> > 这发生在卸载 adaconda 之后。
> 
> 我担心这对SSH有何影响。你知道为什么卸载anaconda会导致问题吗？

Sorry, I haven't replied for a long time. I tried to reinstall vscode, but failed. After I reinstalled windows, I solved the problem. At present, this bug has not been repeated. If I find this rule in the future, I will record it here again.

**Comment by ClintRajaniemi** [comment]
@lindorx 
> Sorry, I haven't replied for a long time. I tried to reinstall vscode, but failed. After I reinstalled windows, I solved the problem. At present, this bug has not been repeated. If I find this rule in the future, I will record it here again.

Ok, so I've been plagued by the issue around uninstalling Anaconda and/or Miniconda with respect to cmd.exe. This was another extension to that problem. Simply, uninstalling Anaconda/Miniconda which had the "conda init" set will, when uninstalled leave a broken autorun key in the registry which causes cmd.exe to open and then immediately close. 

Naturally, reinstalling the OS will fix this, but if you install and remove Anaconda/Miniconda again, it'll happen again. 

I identified the issue and notified the Continuum team of it as well as give some instructions for users who have come across it. 

[Anaconda broken cmd.exe resolution steps](https://github.com/ContinuumIO/anaconda-issues/issues/11702#issuecomment-883825517)

Once I fixed this, the SSH Remote Explorer connected successfully without error.
